pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -233,6 +233,9 @@ func (d *ProjectStatusDescriber) Describe(namespace, name string) (string, error
 			printLines(out, indent, 0, describeMonopod(f, monopod.Pod)...)
 		}
 
+		bcPipelines, _ := graphview.AllImagePipelinesFromBuildConfig(g, graphview.IntSet{})
```

Comment:
ok, i've pushed the updates for "produced by ...."

pending comments on the specific changes, putting this PR on hold until https://trello.com/c/hHx4HrCG/966-integrate-pipeline-stages-into-oc-status gets going enough to determine for sure whether this addition to the existing output is acceptable.